submissions/Ant-C-tech/tiny-js-world/index.js

```diff
@@ -0,0 +1,135 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/Ant-C-tech/a-tiny-JS-world
+   Web app: _put project's github pages URL here_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+const dog = {
+   species: 'dog',
+   name: 'Spike',
+   gender: 'male',
+   legs: 4,
+   hands: 0,
+   saying: 'Woof-Woof!',
+   friends: ['John']
+}
+
+const cat = {
+   species: 'cat',
+   name: 'Tom',
+   gender: 'male',
+   legs: 4,
+   hands: 0,
+   saying: 'Meow-Meow!',
+   friends: ['John']
+}
+
+const woman = {
+   species: 'human',
+   name: 'Jane',
+   gender: 'female',
+   legs: 2,
+   hands: 2,
+   saying: 'Give me your salary!'
+}
+
+const man = {
+   species: 'human',
+   name: 'John',
+   gender: 'male',
+   legs: 2,
+   hands: 2,
+   saying: 'Give me your clothes, your boots and your motorcycle!',
+   friends: ['Spike', 'Tom']
+}
+
+function createGreetingContent(obj) {
+
+   let content = ''
+   let interlocutor = ''
+   
+   switch (true) {
+      case obj.species == 'dog':
+         interlocutor = 'a cat'
+         break;
+      case obj.species == 'cat':
+         interlocutor = 'a dog'
+         break;
+      case obj.species == 'human' && obj.gender == 'male':
+         interlocutor = 'a biker'
+         break;
+      case obj.species == 'human' && obj.gender == 'female':
+         interlocutor = 'my husband'
+         break;
```

Comment:
If we have many cats, or dogs, men and women, they all have similar interlocutors.
Not really realistic and even if so there are no reasons to assign interlocutor just like legs or hands, or species at object definition stage.

`switch (true) { case condition: ... }` is an antipattern that should be avoided.